lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskServiceImpl.java

```diff
@@ -261,7 +261,7 @@ public Task createTask(Task taskToCreate)
         // primary key violation: "UC_EXTERNAL_ID_INDEX_2 ON TASKANA.TASK(EXTERNAL_ID) ...
         String msg = e.getMessage() != null ? e.getMessage().toLowerCase() : null;
         if (msg != null
-            && (msg.contains("violation") || msg.contains("violates"))
+            && (msg.contains("violation") || msg.contains("violates") || msg.contains("verletzt"))
```

Comment:
This looks like a big code smell due to the dependency to the database locale.
Ticket?

Reply:
Yes I also think a ticket would be great. Maybe there is a way to get this done in a better way. I always thought that all DBs have an explicit "UniqueContraintViolation" exception. It seems that is not the case and therefore this workaround is implemented.

Reply:
did you create the ticket?

Reply:
Yes, now i did ;D
https://taskana.atlassian.net/browse/TSK-1231